src/cloudlabs/app.py

```diff
@@ -22,6 +24,17 @@ def create_app(config_name=None):
         config_name = os.getenv('APP_SETTINGS', 'cloudlabs.config.Config')
     app = Flask(__name__)
     app.config.from_object(config_name)
+    # Configure logging
+    # It seems we have to set this to at least as low as the file handler level
+    # (at least for development builds, perhaps not for the actual servers?)
+    app.logger.setLevel(logging.DEBUG)
+    handler = RotatingFileHandler('cloudlabs.log', maxBytes=10000, backupCount=5)
+    handler.setLevel(logging.INFO)  # required only if we want a higher level than the app logger
+    handler.setFormatter(logging.Formatter(
+        '%(asctime)s %(levelname)s: %(message)s '
+        '[in %(pathname)s:%(lineno)d]'
+    ))
+    app.logger.addHandler(handler)
```

Comment:
Can this configuration be done on a separate config file? I've found that useful in the past and it might come in handy as the application grows so we can have separate loggers for different things (e.g. one for deployment related messages, one for user management, etc.). 

Maybe [this](https://stackoverflow.com/questions/11816236/how-to-config-flask-app-logger-from-a-configure-file?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa) and [this](https://gist.github.com/jakubczaplicki/c9077a482c2b195b7f3bcbee19dde441) help.

Reply:
Yes, I agree that we should. I wanted to get something started and then forgot about it. I'll do this, possibly after #72 is merged so we have the new config too.

Reply:
I just reread this and had misunderstood - I was thinking of doing it in `config.py` for each configuration of the Flask app (development, production, ...). Did you have a separate file that controls the logging configuration as in Nammu?

Reply:
Now done similarly to Nammu.